### ISSUE TITLE
[Bug][A11Y][Badge] Icon doesn't resize

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#844]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS 18

### App version

0.17.0

### App build number

175218176

### App build type

beta

### Describe the issue

After text enlargement, the count is correctly resized but not the icon

<img width="321" height="249" alt="Image" src="https://github.com/user-attachments/assets/dcb13032-950f-4e11-99b1-aeb7c6808f0c" />

<img width="319" height="275" alt="Image" src="https://github.com/user-attachments/assets/aa8b4bf1-36a5-4fae-966a-e6090e2b2f11" />

### Expected Behavior

icon should be resized as text

### Accessibility settings

increase text sizes

**Comment by pylapp** [comment]
@pya35 Ce cas montre encore une fois que l'a11y n'est pas prise en compte suffisamment dans les spécifications design, mais on ne s'étonne plus à force.
Je vais apporter une première correction en appliquant une taille de badge donnée si on a un texte grossi .

Cependant dit je prends la plus grosse des tailles disponibles dans le design system, et ça risque d'être insuffisant.
Si c'est le cas je t'invite à faire une carte dédiée avec des suggestions de tailles pour aider l'équipe design.

La taille appliquée est listée ci-dessous selon les thèmes : 

| Orange       | Inverse      | Business Tools | Sosh         | Wireframe    |
|--------------|--------------|----------------|--------------|--------------|
| dimension4xs | dimension4xs | dimension3xs   | dimension4xs | dimension4xs |
| dimension250 | dimension250 | dimension250   | dimension250 | dimension250 |
| 20px         | 20px         | 20px           | 20px         | 20px         |

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.18.0**
- Build number: **1755794658**
- Build tag: **4af4fa4**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.18.0**
- Build number: **1755874772**
- Build tag: **959dc2f**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.18.0**
- Build number: **1756329709**
- Build tag: **193c325**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.19.0**
- Build number: **1758143890**
- Build tag: **8385a6b**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **0.19.0**
- Build number: **1758316731**
- Build tag: **d792364**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/0.19.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **0.19.0**
- Build number: **1758737311**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
